removeDuplicate.js

```diff
@@ -0,0 +1,4 @@
+// Implement a JavaScript function to remove duplicate elements from an array.
+function removeDuplicates(array){
+return array.filter((el, i) => array.indexOf(el) === i)
```

Comment:
When dealing with unique elements in a collection you can alway think of sets, read more about `Set` in javascript [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)